src/Sylius/Bundle/ApiBundle/spec/Serializer/ProductAttributeValueNormalizerSpec.php

```diff
@@ -86,10 +86,13 @@ function it_serializes_product_attribute_select_values(
                     'de_DE' => 'de text4',
                     'zu_ZA' => 'zu text4',
                 ],
+                'uuid5' => [
+                    'fr_FR' => null
```

Comment:
I do not agree, the value should not exist here, not having a null.

Reply:
Lol I test it locally only with your Phpspec change and the test pass, so that proves nothing :dagger: